rss2email/config.py

```diff
@@ -40,6 +40,22 @@
 
 import html2text as _html2text
 
+# These key/value pairs override all values with the same key in all
+# sections, including feed-specific sections. Overrides can be set
+# with the -s/--set command line option.
+OVERRIDES = {}
```

Comment:
Does this have to be a global variable? Can’t we use the normal data path?

Reply:
Good point, made change to correct this.